src/Util/AnnotationFilterExtractorTrait.php

```diff
@@ -89,9 +89,9 @@ private function readFilterAnnotations(\ReflectionClass $reflectionClass, Reader
             $filterClass = $filterAnnotation->filterClass;
             $id = $this->generateFilterId($reflectionClass, $filterClass);
```

Comment:
@mykidok Why not updating `generateFilterId()` directly ?
The class is `@internal` so adding a default argument is not a BC break